test/MUXControlsTestApp/TestInventory.cs

```diff
@@ -34,58 +34,165 @@ static TestInventory()
         {
             Tests = new List<TestDeclaration>()
             {
-#if !BUILD_LEAN_MUX_FOR_THE_STORE_APP
+// #if FEATURE_SCROLLER_ENABLED && FEATURE_COLOR_PICKER_ENABLED && FEATURE_NAVIGATION_VIEW_ENABLED && FEATURE_RATING_CONTROL_ENABLED
+//              {"Leak", typeof(LeakTestPage)},
+//#endif
+
+#if FEATURE_COLOR_PICKER_ENABLED
                 {"ColorPicker", typeof(ColorPickerPage), "ColorPicker.png"},
-                {"Leak", typeof(LeakTestPage)},
+#endif
+
+#if FEATURE_PERSON_PICTURE_ENABLED
```

Comment:
Consider doing this with reflection instead of a bunch of ifdefs. You could create a new attribute [WinUITestPage] (or whatever) then apply that to all the top-level test pages. Then do reflection across the types to find them.

If you don't want to do it, please file a GitHub issue. You can even assign it to me. :)